R/quotation.R

```diff
@@ -1,109 +1,95 @@
-#' Quotation
+#' Defusing R expressions
 #'
 #' @description
 #'
 #' \Sexpr[results=rd, stage=render]{rlang:::lifecycle("stable")}
 #'
-#' Quotation is a mechanism by which an expression supplied as
-#' argument is captured by a function. Instead of seeing the value of
-#' the argument, the function sees the recipe (the R code) to make
-#' that value. This is possible because R [expressions][is_expr] are
-#' representable as regular objects in R:
+#' When a function argument is defused, R doesn't return its value
+#' like it normally would. Instead, it returns the R expression
+#' describing how to make the value as an inert object. These defused
+#' expressions are like blueprints for computing values.
```

Comment:
I think you need a parenthetical here saying something like "(This is also known as quoting, and is what is done by `substitute()`"), i.e. something to sign post to the more experience reader that this is the content that they're looking for.